release-calendar/package.json

```diff
@@ -68,13 +72,15 @@
     "webpack-node-externals": "1.7.2"
   },
   "devDependencies": {
+    "empty-module": "0.0.2",
```

Comment:
Move this and moment-timezone-data-webpack-plugin to dependencies. It's needed there along with other webpack packages for deployment.